values/otel-operator/otel-operator-raw.gotmpl

```diff
@@ -0,0 +1,68 @@
+{{- $v := .Values }}
+{{- $o := $v.apps.otel }}
+resources:
+  - apiVersion: opentelemetry.io/v1alpha1
+    kind: OpenTelemetryCollector
+    metadata:
+      name: otel-collector
+    spec:
+      config: |
+        receivers:
+          otlp:
+            protocols:
+              grpc:
+              http:
+        processors:
+          memory_limiter:
+            check_interval: 1s
+            limit_percentage: 75
+            spike_limit_percentage: 15
+          batch:
+            send_batch_size: 10000
+            timeout: 10s
+        exporters:
+          logging:
+            loglevel: info
+          {{- if $v.apps.tempo.enabled }}
+          otlp:
+            endpoint: tempo-distributor.tempo.svc.cluster.local:4317
+            sending_queue:
+              enabled: true
+              num_consumers: 100
+              queue_size: 10000
+            retry_on_failure:
+              enabled: true
+            tls:
+              insecure: true
```

Comment:
Can we change this flag to `insecure: false`

Reply:
no. but is only in cluster